.claude/commands/release.md

```
@@ -1,186 +1,545 @@
+```
+
+## Step 7: Detect Special Changes
+
+### OpenAPI Changes
+
+If `release.watchFiles.openapi` is configured:
+
+```bash
+OPENAPI_FILE=$(config.release.watchFiles.openapi || "docs/openapi.json")
+
+if git diff --quiet origin/${PROD_BRANCH}..origin/${DEV_BRANCH} -- "${OPENAPI_FILE}"; then
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)